src/Traits/NumeralTypeTrait.php

```diff
@@ -45,110 +45,120 @@ trait NumeralTypeTrait
      * Add this expression to the given expression.
      *
      * @param NumeralType $right
+     * @param bool $insertParentheses
      * @return Addition
      */
-    public function plus(NumeralType $right): Addition
+    public function plus(NumeralType $right, bool $insertParentheses = true): Addition
```

Comment:
The `$insertParentheses` parameter should also be added to the relevant interface (`NumeralType`).

Reply:
fixed in fa15938